roles/kubernetes-apps/cloud_controller/kube-vip-cloud-provider/tasks/main.yml

```diff
@@ -0,0 +1,49 @@
+---
+
+- name: Kube-VIP Cloud Provider  | Check kube-vip settings for kube-vip cloud provider
+  fail:
+    msg: "kube-vip cloud provider require kube_vip_enabled = true and kube_vip_services_enabled = true"
+  when:
```

Comment:
shouldn't this be:

when: not (kube_vip_enabled and kube_vip_services_enabled)